components/PageHeader.vue

```diff
@@ -7,12 +7,35 @@
       {{ title }}
     </h2>
     <div class="date">
-      <span>最終更新 </span>
+      <span>{{ $t('最終更新') }} </span>
       <time :datetime="formattedDate">{{ date }}</time>
     </div>
   </div>
 </template>
 
+<i18n>
+{
+  "ja": {
+    "最終更新": "最終更新"
+  },
+  "en": {
+    "最終更新": "Last update"
+  },
+  "zh-cn": {
+    "最終更新": "最后更新"
+  },
+  "zh-tw": {
+    "最終更新": "前次更新"
```

Comment:
#852 によるとシートが変更されていたので
```suggestion
    "最終更新": "上次更新"
```

Reply:
@himatani これ取り込みお願いします

Reply:
@nard-tech これ私の方で取り込んでも良いですか？

Reply:
@halsk お願いします！